src/account/setup.cfg

```diff
@@ -0,0 +1,2 @@
+[bdist_wheel]
+universal=1
```

Comment:
cloud be removed.

Reply:
entire file?

Reply:
You can keep the file for convenience of future editing.